osu.Game/Rulesets/UI/ModIcon.cs

```diff
@@ -81,19 +92,17 @@ public ModIcon(IMod mod, bool showTooltip = true)
                 {
                     Origin = Anchor.Centre,
                     Anchor = Anchor.Centre,
-                    Colour = OsuColour.Gray(84),
                     Alpha = 0,
                     Font = OsuFont.Numeric.With(null, 22f),
                     UseFullGlyphHeight = false,
-                    Text = mod.Acronym
                 },
-                modIcon = new SpriteIcon
+                modIcon = new OsuSpriteText
```

Comment:
It kind of irks me that we can't use `SpriteIcon` for an icon and we have to use... text instead?

Maybe we could expose the fill mode of the icon from framework so that it fills the smallest dimension rather than fitting to it.